src/core_atmosphere/physics/mpas_atmphys_driver_seaice.F

```diff
@@ -504,21 +509,24 @@ subroutine seaice_to_MPAS(configs,mesh,diag_physics,sfc_input,its,ite)
  character(len=StrKIND),pointer:: sfclayer_scheme
 
  real(kind=RKIND),dimension(:),pointer:: acsnom,acsnow,chs,chs2,cpm,cqs2,qgh,qsfc,grdflx,hfx, qfx,lh,noahres, &
-                                         potevp,sfc_albedo,sfc_emiss,sfcrunoff,snopcx,z0,znt
+                                         potevp,sfc_albedo,sfc_albbck,sfc_emiss,sfcrunoff,snopcx,z0,znt
```

Comment:
for future merge reasons, sfc_albbck should be moved to the line below where you added flqc and flhc; sticking it in the middle of an existing ncar variable list will cause future conflicts

Reply:
sfc_albbck is moved to the line with flqc, flhc